### ISSUE TITLE
PR #126 should have been in 3.0

### ISSUE THREAD

**Issue by tm1000** [xmppjs/ltx#129]
This relates to really old versions of nodejs , specifically 0.12.x. Basically the work in #126 destroys anyone using older versions of node-xmpp-server on 0.12.x because of the use of CONST. The issue is the fact that for instance node-xmpp-core has a dep of ltx ^2.2.0 which means that npm will select 2.8.0 which blows up if you are using 0.12.x.

I understand the need to use new ES6 code but priority should be taken to not blow up older versions. This is why it should have been in 3.0.

```
2018-10-16 13:27 -04:00: lets-chat/node_modules/node-xmpp-server/node_modules/node-xmpp-core/node_modules/ltx/lib/parsers/ltx.js:68
        const lt = data.indexOf('<', pos)
        ^^^^^
SyntaxError: Use of const in strict mode.
```

**Comment by jpablom** [comment]
I have the same issue. I reported an error before (#124) and again the use of es6 it's causes me troubles.
Hope that can be resolve soon. Thanks

**Comment by sonnyp** [comment]
oops sorry, fixed in v2.8.1
